hsds/basenode.py

```diff
@@ -33,7 +33,7 @@
 from .util.k8sClient import getDnLabelSelector, getPodIps
 from . import hsds_logger as log
 
-HSDS_VERSION = "0.7.3"
```

Comment:
Upping the version here lets the REST VOL check it to determine whether to use the new h5path logic or not.